src/zjs_console.c

```diff
@@ -231,10 +229,10 @@ static jerry_value_t console_assert(const jerry_value_t function_obj,
                                     const jerry_value_t argv[],
                                     const jerry_length_t argc)
 {
+    // args: validity[, output]
+    ZJS_VALIDATE_ARGS(Z_BOOL, Z_OPTIONAL Z_ANY);
+
     char message[MAX_STR_LENGTH];
-    if (!jerry_value_is_boolean(argv[0])) {
```

Comment:
Here's one example of where this patch fixed a minor bug - before we weren't checking to make sure there *was* an argv[0] so if a JS user passed no args we would have made an illegal access.